nautobot/docs/release-notes/version-2.2.md

```diff
@@ -18,6 +18,12 @@ New models have been added for software image files and software versions. These
 
 Software versions must be associated to a platform. Software image files must be associated to one software version and may be associated to one or more device types. Devices, inventory items and virtual machines may be associated to one software version to track their current version. See the documentation for [software image file](../user-guide/core-data-model/dcim/softwareimagefile.md) and [software version](../user-guide/core-data-model/dcim/softwareversion.md). There is also a [user guide](../user-guide/feature-guides/software-image-files-and-versions.md) with instructions on how to create these models.
 
+#### Controller models
+
+Controller models have been added to the `dcim` app. A Controller in Nautobot is an abstraction meant to represent network or SDN (Software-Defined Networking) controllers. These may include, but are not limited to, wireless controllers, cloud-based network management systems, and other forms of central network control mechanisms.
+
+For more details, refer to the user guide for a [`Controller` model](../user-guide/core-data-model/dcim/controller.md), a [`ControllerDeviceGroup` model](../user-guide/core-data-model/dcim/controllerdevicegroup.md), or developer documentation for [controllers](../development/core/controllers.md).
+
```

Comment:
#5455 added a different version of this content earlier in the file - let's pull in the branch and make sure they're consolidated.